crates/wasmtime/src/runtime/externals/table.rs

```diff
@@ -188,6 +188,8 @@ impl Table {
                         ty => unreachable!("not a top type: {ty:?}"),
                     }
                 }
+
+                runtime::TableElement::ContRef(_c) => todo!(), // TODO(dhil): Required for the embedder API.
```

Comment:
Like above for globals the panic here is fine for now but this'd be good to tag with a `// FIXME` pointing to an issue